### PR TITLE
fix: error ps1.5.3.1 carrier name + logo Alma

### PR DESCRIPTION
**Pull request by Benjamin-Freoua-Alma**

### CODE REVIEW

File: alma/lib/Model/CarrierData.php [hunk 1]
```diff
@@ -25,6 +25,7 @@
 namespace Alma\PrestaShop\Model;
 
 use Carrier;
```
Comment (Francois-Gomis) [inline alma/lib/Model/CarrierData.php:27]: Do you use Carrier ?

**Review by syjust-alma — COMMENTED** [review]
Where alma.gif is used?

**Review by ClaireAlma — APPROVED** [review]

**Comment by Benjamin-Freoua-Alma** [comment]
> Where alma.gif is used?

It's for the tab in Prestashop 1.5
Capture d’écran 2023-01-19 à 10 33 34
@syjust-alma

File: alma/lib/Model/CarrierData.php (hunk 1 above)
Comment (Benjamin-Freoua-Alma) [reply alma/lib/Model/CarrierData.php]: @Francois-Gomis You're right 😅 I remove its for the fix

**Review by syjust-alma — APPROVED** [review]

**Pull request merged**